src/box/alter.cc

```diff
@@ -5868,68 +5868,35 @@ on_replace_dd_func_index(struct trigger *trigger, void *event)
 	return 0;
 }
 
-struct trigger alter_space_on_replace_space = {
-	RLIST_LINK_INITIALIZER, on_replace_dd_space, NULL, NULL
-};
+TRIGGER(alter_space_on_replace_space, on_replace_dd_space);
 
```

Comment:
Nit: Let's remove empty lines between triggers? This would look better IMO:

```C
TRIGGER(alter_space_on_replace_space, on_replace_dd_space);
TRIGGER(alter_space_on_replace_index, on_replace_dd_index);
TRIGGER(on_replace_truncate, on_replace_dd_truncate);
```

Reply:
Sure.